.semaphore/versions.rb

```diff
@@ -3,10 +3,10 @@
 # Update it in appsignal-elixir, then run script/generate_ci_matrix to
 # redownload it.
 VERSIONS = {
-  "25.2" => ["1.14.3", "1.13.4"],
-  "24.3" => ["1.14.3", "1.13.3", "1.12.3", "1.11.4"],
-  "23.3" => ["1.14.3", "1.13.3", "1.12.3", "1.11.4", "1.10.4"],
-  "22.3" => ["1.13.3", "1.12.3", "1.11.4", "1.10.4", "1.9.4"],
-  "21.3" => ["1.11.4", "1.10.4", "1.9.4"],
-  "20.3" => ["1.9.4"]
+  "25.2" => {elixir: ["1.14.3", "1.13.4"], phoenix: ["~> 1.7.0"]},
+  "24.3" => {elixir: ["1.14.3", "1.13.3", "1.12.3", "1.11.4"], phoenix: ["~> 1.7.0"]},
+  "23.3" => {elixir: ["1.14.3", "1.13.3", "1.12.3", "1.11.4", "1.10.4"], phoenix: ["~> 1.6.0"]},
+  "22.3" => {elixir: ["1.13.3", "1.12.3", "1.11.4", "1.10.4", "1.9.4"], phoenix: ["~> 1.6.0"]},
+  "21.3" => {elixir: ["1.11.4", "1.10.4", "1.9.4"], phoenix: ["~> 1.6.0"]},
+  "20.3" => {elixir: ["1.9.4"], phoenix: ["~> 1.6.0"]}
```

Comment:
Do we want to test any Elixir version against both Phoenix versions?

Reply:
The main reason for this is to resolve ht problem introduced in #65, where code that’s being used wasn’t run on CI. The code supports adding multiple Phoenix versions to run on the same Elixir, but I haven’t added any versions where that’s the case. We might decide to in the future.